flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerServicesConfiguration.java

```diff
@@ -78,13 +78,13 @@
 
 	private final long timerServiceShutdownTimeout;
 
-	private final LocalRecoveryConfig.LocalRecoveryMode localRecoveryMode;
+	private final boolean localRecovery;
```

Comment:
Maybe we could rename this field to `isLocalRecoveryEnabled`

Reply:
I would agree to `localRecoveryEnabled` because IIRC the `is` should not be part of boolean field names, only of the accessor method name.